src/Trixi.jl

```diff
@@ -219,7 +219,7 @@ export initial_condition_eoc_test_coupled_euler_gravity,
 
 export cons2cons, cons2prim, prim2cons, cons2macroscopic, cons2state, cons2mean,
        cons2entropy, entropy2cons
-export density, pressure, density_pressure, velocity, global_mean_vars,
+export density, velocity, pressure, density_pressure, velocity, global_mean_vars,
```

Comment:
```suggestion
export density, pressure, density_pressure, velocity, global_mean_vars,
```

Sorry, I just saw that `velocity` was already exported in the same line.